bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
```

Comment:
It's only used for `storm jar` and `storm sql`, but showing them to `print_command` would be better. I'll address this.
Thanks for the suggestion.